projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -45,6 +45,14 @@ public function reregister_menu_items() {
 			remove_menu_page( 'link-manager.php' );
 		}
 
+		/**
+		 * Action fired after the Admin_Menu is registered,
+		 * which allows for further manipulation.
+		 *
+		 * @since-jetpack 10.2
+		 */
+		do_action( 'after_admin_menu_registered', $this );
```

Comment:
Can you add this filter to Domain_Only_Admin_Menu as well (for consistency reasons)? Domain_Only_Admin_Menu is a direct child of Base_Admin_Menu. :)

Reply:
Good point! Done!